docker-compose-prod.yml

```diff
@@ -0,0 +1,32 @@
+# this file is the list of commands that can be used to run apps on docker rather than typing all of the commands individually
+version: "3" 
+services: 
+    api:
+    image: # the name of the dockerhub repository image will go here 
```

Comment:
this code is copied from the original docker-compose file and updated for productions with key changes such as removing volume line and changing env variables